src/lib/tree/node.ts

```diff
@@ -0,0 +1,69 @@
+/**
+ * @license
+ * Copyright Google LLC All Rights Reserved.
+ *
+ * Use of this source code is governed by an MIT-style license that can be
+ * found in the LICENSE file at https://angular.io/license
+ */
+
+import {
+  ContentChildren,
+  Directive,
+  Input,
+  QueryList
+} from '@angular/core';
+import {
+  CdkNestedTreeNode,
+  CdkTreeNodeDef,
+  CdkTreeNode,
+} from '@angular/cdk/tree';
+import {MatTreeNodeOutlet} from './outlet';
+
+/**
+ * Wrapper for the CdkTree node with Material design styles.
+ */
+// TODO(tinayuangao): use mixinTabIndex
+@Directive({
+  selector: 'mat-tree-node',
+  exportAs: 'matTreeNode',
+  host: {
+    '[attr.role]': 'role',
+    'class': 'mat-tree-node',
+    'tabindex': '0',
+  },
+  providers: [{provide: CdkTreeNode, useExisting: MatTreeNode}]
+})
+export class MatTreeNode<T> extends CdkTreeNode<T> {
+  @Input('matTreeNode') data: T;
```

Comment:
@tinayuangao `MatTreeNode<T>` has Input `matTreeNode`, but the equivalent `CdkTreeNode<T>` does not have an Input called `cdkTreeNode`. This means that the same code to create a mat-tree isn't working when switched to cdk-tree

Reply:
This should be removed. Thanks.